### DISCUSSION TITLE
Show Value Above Stacked Bar Graph

### DISCUSSION THREAD

**Discussion by crockk** [grafana/grafana#38204] [Dashboards, Panels & Visualizations]
Hi everyone, 

I am working on a project that includes a number of bar graphs with aggregate data, so I have been using stacked bar graphs to show the sources and values. It would be extremely useful to display the aggregated sum of each individual bar on top of the respective bar. I don't think this is very high priority, but I can't imagine it being too difficult to implement, as the value for the bar graph is already there. This would greatly increase readability for bar graphs overall if this was a global option (eg. Display individual bar values Y/N).

Here's an example of what I mean:

![image](https://user-images.githubusercontent.com/58015900/105226557-fbfbf880-5b14-11eb-964a-a8c90c37d150.png)

**Comment by crockk** [comment]
I believe this was added with the Bar Chart visualization in Grafana v8.0: https://grafana.com/docs/grafana/latest/panels/visualizations/bar-chart/#show-values

I haven't been able to upgrade my Grafana instance yet to test, but I will close this issue for now unless I am mistaken.

**Comment by leeoniya** [comment]
yes, though for your use case you'll have to wait until we have bar stacking working in BarChart panel (8.1-8.3)

**Comment by crockk** [comment]
oh ok, I'll rename and reopen this issue until then.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Reply by Aesculapius** [reply]
Has this been implemented meanwhile? I'm running v8.5.1 and can't find this feature. In need of it to summarize energy usage where low/high tarif are combined.

![image](https://user-images.githubusercontent.com/617061/169145063-bef19f74-c593-4b37-9cb0-9ce080b048cf.png)

**Reply by waywit** [reply]
Hi, I also did not find such functionality :-( 

I think we have the same issue here... I have two electricity tariff with high and low price, for each I have the corresponding power consumption. Now I want to visualize the added (full) consumption in this bar chart, which works, but only the individual consumption values are shown, but I want also the sum of these values representing the consumption of a day. I also have the combined consumption values in a third time series, but this does not help, because then all three consumption values were stacked ;-)
Thanks...

**Reply by thoro** [reply]
I have the same issue (also electricity related) and would like to display the value on a non stacked graph above in vertical orientation.

see: https://github.com/grafana/grafana/discussions/59632

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This seems to be implemented. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution to what you're looking for and would like to request a feature.
